docker-compose.yml

```diff
@@ -1,13 +1,14 @@
-version: '2.4'
+version: '2.5'
```

Comment:
Is an upgrade to the 2.5 compose file format necessary for these changes?

Reply:
I thought so. or at least 2.4.1

obviously functionality doesn't change, but the file organization is different enough that it would cause breaking changes if devs weren't aware.